front/src/config/i18n/en.json

```diff
@@ -2052,7 +2052,9 @@
       "paused": "Paused",
       "exited": "Exited",
       "dead": "Dead"
-    }
+    },
+    "batteryLevel": "Battery level alert",
+    "batteryLevelDescription": "At 09:00 AM, only on Saturday, a message will be sent to all admins as soon as the battery level drops below the chosen percentage"
```

Comment:
Every Saturday at 9:00 am, a message will be sent to all administrators if a device's battery level falls below the chosen threshold.